training/distributed_training/pytorch/data_parallel/rnnt/RNNT_notebook.ipynb

```diff
@@ -0,0 +1,371 @@
+{
```

Comment:
<p>Grammar: <code>SageMaker by default use</code>  -&gt; <code>uses</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Can you link to the exact file of the mlcommons example that was adapted for this notebook?</p>

Reply:
added thanks!